### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Fene-87**
### In this milestone, I use preprocessed HTML file with embedded Ruby code. I have accomplished the following:
- Implement the design from the sneak peek wireframes.
- Use methods in the models do get data.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,17 @@
...
+  <h1>Here is a list of all users</h1>
+  <ul class='users_list'>
+    <% @users.each do |user| %>
+      <li class='user_card'>
+        <div class='user_img'>
+          <img src= <%= user.photo %> alt= <%= user.name %> />
+        </div>
+
+        <div class='user_stuff'>
+          <p class='user_name'><%= user.name %></p>
```
Comment (franclobo) [inline app/views/users/index.html.erb]: - [ ] The name should be a link that allows to see the user information, please 🙏🏽 refactor your code to make it possible 👇🏽 
```ruby
<p class='user_name'><%= link_to user.name, user_path(user) %></p>
```

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,37 @@
...
+  </div>
+
+  <ul class='posts_list'>
+    <% if @user.posts.empty? %>
+      <li class='post_card'><%= @user.name %> has no posts</li>
+    <% end %>
+
+    <% @user.recent_posts.each_with_index do |post, index| %>
+      <li class='post_card'>
+        <h4 class='post_header'>Post #<%= index + 1 %></h4>
```
Comment (franclobo) [inline app/views/users/show.html.erb]: - [ ] The post title should be a link that allows see the comments of the post, please 🙏🏽 fix that issue to see the comments 👇🏽 

```ruby
<h4 class='post_header'><%= link_to post.title, user_post_path(post) %></h4>
```

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,39 @@
...
+      <p class='user_name'><%= @user.name %></p>
+      <p class='posts_no'>Number of Posts: <%= @user.postscounter || 0 %></p>
+    </div>
+  </div>
+
+  <ul class='posts_list'>
+    <% @user.posts.each_with_index do |post, index|%>
+      <li>
+        <div class='post_card'>
+          <h4 class='post_header'>post #<%= index + 1%></h4>
```
Comment (franclobo) [inline app/views/posts/index.html.erb]: - [ ] The post title should be a link that allows see the comments of the post, please 🙏🏽 fix that issue to see the comments 👇🏽 

```ruby
<h4 class='post_header'><%= link_to post.title, user_posts_path(post) %></h4>
```

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Fene-87, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️
- Correct Git flow ✔️ 
- Good UI ✔️  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (Fene-87) [reply app/views/users/index.html.erb]: Done

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (Fene-87) [reply app/views/posts/index.html.erb]: Done

File: app/views/users/show.html.erb (hunk 2 above)
Comment (Fene-87) [reply app/views/users/show.html.erb]: Done

**Review by Davidosky007 — APPROVED** [review]
Hi @Fene-87 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Well written PR 👍 
* Good looking readme 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**